views/index.erb

```diff
@@ -0,0 +1,17 @@
+<link rel="stylesheet" type="text/css" href="/stylesheets/chitter.css" />
+<body>
+  <div class="header">
+    <% if @user %>
```

Comment:
Having these if statements feels ok to me because it's the logic of what's displayed to the user